metricbeat/module/system/process_summary/process_summary.go

```diff
@@ -109,6 +112,7 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		"stopped":  summary.stopped,
 		"zombie":   summary.zombie,
 		"unknown":  summary.unknown,
+		"dead":     summary.dead,
```

Comment:
You will need to add this new field to `metricbeat/module/system/process_summary/_meta/fields.yml`, after adding it you will also need to run `make update`.

Reply:
Done in e4c9bab8e90c4151813763c544210eb2a08f491e